index.html

```diff
@@ -105,11 +105,11 @@
 
       function log(text, style = "") {
         var date = new Date();
-        var timestamp = [date.getHours(), date.getMinutes(), date.getSeconds()].join(":");
+        var timestamp = [('0'+date.getHours()).slice(-2), ('0'+date.getMinutes()).slice(-2), ('0'+date.getSeconds()).slice(-2)].join(":");
```

Comment:
Then you can do `[date.getHours(), date.getMinutes(), date.getSeconds()].map(pad).join(":")`.